x/mint/types/minter.go

```diff
@@ -83,19 +73,11 @@ func ValidateMinter(minter Minter) error {
 }
 
 func calcMintedTokens(m Minter) sdkmath.Uint {
-	if m.NormTimePassed.GTE(MonthsInFormula) {
-		fixedMonthsPeriod := sdkmath.NewUint(m.NormTimePassed.Sub(MonthsInFormula).TruncateInt().Uint64())
-		fixedMonthsTokens := fixedMonthsPeriod.Mul(FixedMintedAmount)
-		calculatedTokensByIntegral := CalcTokensByIntegral(MonthsInFormula).Sub(CalcTokensByIntegral(NormOffset))
-
-		return calculatedTokensByIntegral.Add(fixedMonthsTokens)
-	} else {
-		return CalcTokensByIntegral(m.NormTimePassed).Sub(CalcTokensByIntegral(NormOffset))
-	}
+	return CalcTokensByIntegral(m.NormTimePassed).Sub(CalcTokensByIntegral(NormOffset))
```

Comment:
I am afraid this is incorrect since it calculates only the minted amount following the updated formula ... whereas the really minted tokens differ ... and that would break the equality check in the other code

Reply:
As discussed, I confirm that ValidateMinter is not called during the normal block lifecycle, it is only called when validating genesis. 
It is safe to remove this check